client/src/mapView.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { Map, TileLayer } from 'react-leaflet';
+import ReactModal from 'react-modal';
+
+import { tileSet, SFGeo, zoomLevel, mapAttribution } from '../mapconfig';
+import MarkerCollection from './markerCollection';
+import fetchParklets from '../utils/fetchParklets';
+
+const PARKLETS_ENDPOINT = 'https://data.sfgov.org/resource/6a7x-cttf.json';
+
+export default class MapView extends Component {
```

Comment:
Export pattern is a bit inconsistent. Inline export here while other modules export at the bottom. I prefer bottom export but either way works as long as they're consistent.